tests/converter_optimizer_test.py

```diff
@@ -9,7 +9,6 @@
 from tinynn.converter import TFLiteConverter
 from tinynn.converter.schemas.tflite import schema_generated as tflite
 
-
```

Comment:
We use the formatter `black` in the project. Please refer to [the contribution guidelines](https://github.com/alibaba/TinyNeuralNetwork/blob/main/CONTRIBUTING.md#formatting) for more details.